### PR TITLE
Tweak open source documentation

### PR DESCRIPTION
**Pull request by subsymbolic**
Task/Issue URL: https://app.asana.com/0/856498667320406/1107787706804409
Tech Design URL: N/A

**Description**:
Update README and contributions files to match iOS tweaks.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,16 +4,27 @@ Thank you for taking the time to contribute to DuckDuckGo! :sparkles:
 
 We are pleased to open up the project to you - our community. How can you contribute?
 
-## Share feedback and bug reports
-See [README.md](README.md)
+## Share feedback
+Contact us at https://duckduckgo.com/feedback if you have feedback, questions or want to chat. You can also use the feedback form embedded within our Mobile App - to do so please navigate to Settings and select "Send Feedback.
```
Comment (CDRussell) [inline CONTRIBUTING.md]: Missing a closing `"`.

And it's `Leave Feedback` in the Android app.

File: README.md [hunk 2]
```diff
@@ -18,15 +14,13 @@ Alternatively, if you already have the project checked out, you can initialize t
...
+## Discuss
 
-We want our app to be as stable as possible thus your bug reports are immensely valuable. When reporting bugs let us know the:
-* App version
-* Device make and model
-* Android version
-* Steps to reproduce the bug
-* Expected behavior
-* Actual behavior
+Contact us at https://duckduckgo.com/feedback if you have feedback, questions or want to chat. You can also use the feedback form embedded within our Mobile App - to do so please navigate to Settings and select "Send Feedback".
```
Comment (CDRussell) [inline README.md]: As above, `Leave Feedback`

**Comment by subsymbolic** [comment]
@bwaresiak over to you for review.

**Comment by subsymbolic** [comment]
Thanks for the review :)

**Pull request merged**